pkg/storaged/lvol-tabs.jsx

```diff
@@ -396,6 +525,117 @@
     });
 }
 
+const StructureDescription = ({ client, lvol }) => {
+    const struct = lvol.Structure;
+
+    if (!struct)
+        return null;
+
+    let status = null;
+    const status_code = client.lvols_status[lvol.path];
+    if (status_code == "partial") {
+        status = _("This logical volume has lost some of its physical volumes and can no longer be used. You need to delete it and create a new one to take its place.");
+    } else if (status_code == "degraded") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230912-121317-ed28e52d-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)